app/src/app/pages/plando-page/plando-page.component.ts

```diff
@@ -139,6 +139,15 @@ export class PlandoPageComponent implements OnInit, OnDestroy {
     return { invalidPlandoItem: { value: control.value } };
   }
 
+  private perCheckTypeValidator(checkType: CheckType): ValidatorFn {
```

Comment:
Could be moved to custom.validators.ts so this page component can breath a bit more